docs/source/name_inference.rst

```diff
@@ -197,6 +197,8 @@ If you don't see an operation listed here, but it would help your use case, plea
    :meth:`Tensor.sigmoid_`,None
    ":meth:`Tensor.sign`, :func:`torch.sign`",:ref:`keeps_input_names-doc`
    :meth:`Tensor.sign_`,None
+   ":meth:`Tensor.sgn`, :func:`torch.sgn`",:ref:`keeps_input_names-doc`
```

Comment:
You can revert this if you like since the native_function entries don't support named tensors and it's untested.

Reply:
synced offline - not removing it as other unary ops have their entry in this list as well.